### PR TITLE
move global args to be applied to version call

### PR DESCRIPTION
**Pull request by ryanhagen**
**Description**
Executing `helm version` will check the server side namespace `kube-system`. In my particular use case of skaffold, I will not have access to `kube-system`, therefore, global variable `--tiller-namespace` will have to be applied to the `helm version` command, hence this PR.

### CODE REVIEW

**Comment by ryanhagen** [comment]
@googlebot I fixed it.

File: pkg/skaffold/deploy/helm.go [hunk 1]
```diff
@@ -220,9 +220,9 @@ func (h *HelmDeployer) Render(context.Context, io.Writer, []build.Artifact, []La
 
 // exec executes the helm command, writing combined stdout/stderr to the provided writer
 func (h *HelmDeployer) exec(ctx context.Context, out io.Writer, useSecrets bool, args ...string) error {
+	args = append(args, h.Flags.Global...)
```
Comment (tejal29) [inline pkg/skaffold/deploy/helm.go:223]: @ryanhagen please add a another test case where flags is not empty here https://github.com/GoogleContainerTools/skaffold/blob/49888e0abdd6249c18fdb311970a1d7734f48b1a/pkg/skaffold/deploy/helm_test.go#L734

Comment (ryanhagen) [reply pkg/skaffold/deploy/helm.go:223]: See below commit

**Comment by tejal29** [comment]
@ryanhagen the travis job is failing due to lint errors. can you please fix ?

**Comment by briandealwis** [comment]
I don't think this PR is necessary as #4294 causes Skaffold to now use `helm version --client` to avoid contacting the cluster.

**Comment by tejal29** [comment]
@ryanhagen Can we close this PR since #4294  is in as @briandealwis mentioned?

**Comment by ryanhagen** [comment]
@tejal29 sounds good to me.
